lib/i18n_routable.rb

```diff
@@ -1,6 +1,5 @@
-require 'rails/all'
+require 'action_controller/railtie'
```

Comment:
The reason I had to change this was because requiring rails/all would include ActiveRecord which would try to find database.yml. I ended up with these errors:
```
Failure/Error: Unable to find matching line from backtrace
     ActiveRecord::ConnectionNotEstablished:
       No connection pool for ActiveRecord::Base
```